src/ontology/maxo-edit.owl

```diff
@@ -13628,4 +13815,5 @@ AnnotationAssertion(dcterms:contributor <http://purl.obolibrary.org/obo/MAXO_003
 AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/MAXO_0035123> <https://orcid.org/0000-0001-7941-2961>)
 
 
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/MAXO_0010362> "cystoscopy")
```

Comment:
Adding this term revealed the problem that [cystoscopy](http://purl.obolibrary.org/obo/MAXO_0001185) and [cystography procedure](http://purl.obolibrary.org/obo/MAXO_0010362) share the same exact synonym "cystoscopy". Advice?

Reply:
cystography is not the same as cystoscopy and the synonym is wrong. I will remove it and make another PR

Reply:
I fixed it now.